apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOptimize/Components/ArtifactSetConfig.tsx

```diff
@@ -94,10 +95,10 @@ export default function ArtifactSetConfig({ disabled }: { disabled?: boolean, })
         </Typography>
         <Stack spacing={1}>
           <Typography>{t`artSetConfig.setEffCond`} <SqBadge color={artifactCondCount ? "success" : "warning"}>{artifactCondCount} {t<string>("artSetConfig.enabled")}</SqBadge></Typography>
-          <Typography>{t`sheet:2set`} <SqBadge color="success">{allow2} <FontAwesomeIcon icon={faChartLine} className="fa-fw" /> {t<string>("artSetConfig.allowed")}</SqBadge>{!!exclude2 && " / "}{!!exclude2 && <SqBadge color="secondary">{exclude2} <FontAwesomeIcon icon={faBan} className="fa-fw" /> {t<string>("artSetConfig.excluded")}</SqBadge>}</Typography>
-          <Typography>{t`sheet:4set`} <SqBadge color="success">{allow4} <FontAwesomeIcon icon={faChartLine} className="fa-fw" /> {t<string>("artSetConfig.allowed")}</SqBadge>{!!exclude4 && " / "}{!!exclude4 && <SqBadge color="secondary">{exclude4} <FontAwesomeIcon icon={faBan} className="fa-fw" /> {t<string>("artSetConfig.excluded")}</SqBadge>}</Typography>
-          <Typography>{t`artSetConfig.2rainbow`} <SqBadge color={allowRainbow2 ? "success" : "secondary"}><FontAwesomeIcon icon={allowRainbow2 ? faChartLine : faBan} className="fa-fw" /> {allowRainbow2 ? t<string>("artSetConfig.allowed") : "Excluded"}</SqBadge></Typography>
-          <Typography>{t`artSetConfig.4rainbow`} <SqBadge color={allowRainbow4 ? "success" : "secondary"}><FontAwesomeIcon icon={allowRainbow4 ? faChartLine : faBan} className="fa-fw" /> {allowRainbow4 ? t<string>("artSetConfig.allowed") : "Excluded"}</SqBadge></Typography>
+          <Typography>{t`sheet:2set`} <SqBadge sx={{ display: "inline-flex", gap: 0.5 }} color="success">{allow2} <ShowChartIcon {...iconInlineProps} /> {t<string>("artSetConfig.allowed")}</SqBadge>{!!exclude2 && " / "}{!!exclude2 && <SqBadge sx={{ display: "inline-flex", gap: 0.5 }} color="secondary">{exclude2} <BlockIcon {...iconInlineProps} /> {t<string>("artSetConfig.excluded")}</SqBadge>}</Typography>
```

Comment:
Can store the SqBadge sx in a const for easy reuse/future modification

Reply:
those can all be removed